src/models.py

```diff
@@ -15,6 +16,7 @@ class User(Base):
     id = Column(Integer, primary_key=True)  # noqa: A003
     username = Column(String, nullable=False, unique=True)
     password_hash = Column(String(64), nullable=False)
+    user_session = relationship("UserSession", backref="user_identification")
```

Comment:
Mypy gives you an error here because he cannot guess type of `user_session` from sqlachemy's `relationship` function. And plugin for sqlalchemy cannot do this as well.

So to fix this error we have to provide a type for `user_session` by hand, like that:
```python
user_session: UserSession = ...
```

After that mypy will complain about our `UserSession` type we provided, because `UserSession` class is defined below `User` class. And to make mypy understand this type correctly we have to add this import as a first import in `models.py`:
```python
from __future__ import annotations
```

Also we need to specify what our relationship is - is it related to one item in db or many? We can do this by providing `uselist=False` as an argument to `relationship` function, like that:
```python
... = relationship(..., uselist=False)
```

**BUT**, seems like we don't use this relationship field in our code at all, so I think we should just remove this line. That's the easiest and actually the most correct way to fix this error now :)